dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/utils/ConfigValidationUtils.java

```diff
@@ -304,29 +306,35 @@ public static URL loadMonitor(AbstractInterfaceConfig interfaceConfig, URL regis
         AbstractConfig.appendParameters(map, monitor);
         AbstractConfig.appendParameters(map, application);
         String address = null;
-        String sysaddress = System.getProperty("dubbo.monitor.address");
-        if (sysaddress != null && sysaddress.length() > 0) {
-            address = sysaddress;
+        String sysAddress = System.getProperty(DUBBO_MONITOR_ADDRESS);
+        if (sysAddress != null && sysAddress.length() > 0) {
+            address = sysAddress;
         } else if (monitor != null) {
             address = monitor.getAddress();
         }
-        if (ConfigUtils.isNotEmpty(address)) {
+        String protocol = monitor == null ? DUBBO_PROTOCOL : monitor.getProtocol();
```

Comment:
Set protocol to DUBBO here will cause loading default dubbo monitor when the application didnot have any monitor config.

Reply:
`null` should be returned in this scene

Reply:
Thanks, I'll fix it.